lib/perl/Genome/Model/ClinSeq.pm

```diff
@@ -249,6 +233,19 @@ sub _resolve_reference {
 sub __errors__ {
     my $self   = shift;
     my @errors = $self->SUPER::__errors__;
+
+    for my $model (qw(wgs_model exome_model)) {
+        if ($self->$model) {
+            unless ($self->$model->isa('Genome::Model::SomaticVariation') || $self->$model->isa('Genome::Model::SomaticValidation')) {
```

Comment:
After meeting with @jasonwalker80 we decided that the switch to UR::Role is something we would tackle outside of this PR.